clearwater-queue-manager.root/usr/share/clearwater/infrastructure/alarms/clearwater_queue_manager_alarms.json

```diff
@@ -84,6 +84,68 @@
                     "action": "Check that the shared configuration is valid. If it isn't, correct it and re-upload it. If the alarm still doesn't clear, contact your support representative."
                 }
             ]
+        },
+        {
+            "name": "GLOBAL_CHRONOS_GR_CONFIG_RESYNCHING",
+            "index": 9003,
+            "cause": "DATABASE_INCONSISTENCY",
+            "levels": [
+                {
+                    "severity": "CLEARED",
+                    "description": "etcd: Chronos GR configuration is now consistent across the deployment.",
+                    "details": "All Chronos GR configuration has been synchronized across the deployment.",
+                    "cause": "Chronos GR configuration is now consistent across the deployment.",
+                    "effect": "Changes to Chronos GR configuration have been picked up by all nodes in the deployment.",
+                    "action": "No action."
+                },
+                {
+                    "severity": "MINOR",
+                    "description": "etcd: Chronos GR configuration is being synchronized across the cluster.",
+                    "details": "Chronos GR configuration is being synchronized across the cluster. This involves restarting the Chronos processes on individual nodes.",
+                    "cause": "A change has been made to Chronos GR configuration.",
+                    "effect": "Changes to Chronos GR configuration are synchronized across the cluster and processes are restarted to pick up the changes.",
+                    "action": "Monitor this alarm, and wait for it to clear. If the alarm doesn't clear in a timely fashion, contact your support representative."
+                },
+                {
+                    "severity": "CRITICAL",
+                    "description": "etcd: A Chronos node has failed to resync successfully after receiving new Chronos GR configuration.",
+                    "details": "A Chronos node has failed to resync successfully after receiving new Chronos GR configuration. Check that the Chronos GR configuration is valid, and the node is alive.",
+                    "cause": "A change has been made to Chronos GR configuration which has led to the Chronos node not being able to successfully resync.",
+                    "effect": "Timer redundancy may have been lost",
```

Comment:
This effect isn't going to be particularly clear to users. Using `may have been lost` is also weaselly. I'm also not clear at the moment that this is a critical problem. We really mean 'geographic redundancy' may have been lost, right? - we have not lost local redundancy.

I think we should really be spelling out the impact here more clearly at a solution level - e.g. "Geographic redundancy of timers has been lost. In failure scenarios, this may prevent the subscriber registrations, and of offline billing records being kept up to date."